openapi/partner-connect-2.0.yaml

```diff
@@ -477,8 +478,15 @@ components:
           example: "Oracle"
         service_principal_id:
           type: string
-          description: The UUID (username) of the service principal identity that a partner product can use to call Databricks APIs. Note the format is different from the databricks_user_id field in user_info.
+          description: The UUID (username) of the service principal identity that a partner product can use to call Databricks APIs. Note the format is different from the databricks_user_id field in user_info. If empty, no service principal was created
           example: "a2a25a05-3d59-4515-a73b-b8bc5ab79e31"
+        connection_scope:
+          type: string
+          description: The scope of users that can use this connection. Workspace means all users in the same workspace. User means only the user creating it.
+          enum:
+            - workspace
+            - user
+          example: workspace
```

Comment:
Minor issue: is the payload update live in prod?  @yuanchaoma-db

Reply:
It should be live as of yesterday afternoon-ish